vnext/Microsoft.ReactNative/ReactNativeHost.cpp

```diff
@@ -76,6 +76,7 @@ void ReactNativeHost::ReloadInstance() noexcept {
   legacySettings.UseFastRefresh = m_instanceSettings.UseFastRefresh();
   legacySettings.UseLiveReload = m_instanceSettings.UseLiveReload();
   legacySettings.UseWebDebugger = m_instanceSettings.UseWebDebugger();
+  legacySettings.DebuggerPort = m_instanceSettings.DebuggerPort();
 
   Mso::React::ReactOptions reactOptions{};
```

Comment:
>Mso::React [](start = 2, length = 10)

I know that this isn't new in this PR but...why? :)

Reply:
I think I already answered your question in another PR. :) The vnext RNW code is originated from Office code and we did not change/refactor all code yet.
More specifically, the last set of code that we moved from Office was concerned the RN Instance management and it allowed us to run the RN in a background thread and to avoid thread blocking.
Feel free to ping me offline to discuss it. 

---
In reply to: [397689085](https://github.com/microsoft/react-native-windows/pull/4373#discussion_r397689085) [](ancestors = 397689085)